### PR TITLE
fix qmake linux build, and fix the "Quick" example

### PR DESCRIPTION
**Pull request by micpub**

### CODE REVIEW

File: examples/quick/ApplicationWindow.qml [hunk 1]
```diff
@@ -43,7 +43,13 @@ FramelessApplicationWindow {
         }
         return FramelessUtils.defaultSystemLightColor;
     }
-    onClosing: Settings.saveGeometry(window)
+    Connections {
```
Comment (wangwenx190) [inline examples/quick/ApplicationWindow.qml:46]: Why is this change necessary?

File: qmake/core.pri [hunk 2]
```diff
@@ -64,6 +64,9 @@ win32 {
 }
 
 unix:!macx {
+    CONFIG += link_pkgconfig
+    PKGCONFIG += gtk+-3.0 xcb
```
Comment (wangwenx190) [inline qmake/core.pri:68]: Please remove "xcb", it's now not required anymore.

**Review by wangwenx190 — CHANGES_REQUESTED** [review]

**Comment by wangwenx190** [comment]
OK, thanks! But I still want to know why you change the QtQuick demo.

**Comment by micpub** [comment]
i've wrote about it above but let me write it again:

example does not start - on console this can be read: 
`
<WARNING> QQmlApplicationEngine failed to load component
<WARNING> qrc:/Demo/HomePage.qml:28:5: Type Window unavailable
<WARNING> qrc:/Demo/Window.qml:46:5: "FramelessWindow.onClosing" is not available in org.wangwenx190.FramelessHelper 1.0.
<CRITICAL> The QML engine failed to create component: QUrl("qrc:/Demo/HomePage.qml")
`
using Connections element the example doest start-up correctly and works as it should.

**Comment by wangwenx190** [comment]
Could you try to add the following line below the referred line to see if the workaround can be removed?

```cpp
qmlRegisterRevision<QWindow, 254>(QUICK_URI_FULL);
```

https://github.com/wangwenx190/framelesshelper/blob/main/src/quick/framelessquickmodule.cpp#L103

**Comment by micpub** [comment]
it doesnt help at all, still gives error:

`qrc:/Demo/Window.qml:46:5: "FramelessWindow.onClosing" is not available in org.wangwenx190.FramelessHelper 1.0.`

**Comment by wangwenx190** [comment]
OK, thanks for your PR!

**Pull request merged**